core/new-gui/src/app/workspace/service/workflow-graph/model/joint-graph-wrapper.ts

```diff
@@ -202,6 +203,16 @@ export class JointGraphWrapper {
     return jointLinkChangeStream;
   }
 
+  public getJointOperatorCellPostion(operatorID: string): number[] | undefined {
+    const operatorCell = this.jointGraph.getCell(operatorID);
+    const position: number[] = [];
+    if (operatorCell.isElement()) {
+      const result: dia.Element  = new dia.Element(operatorCell.attributes);
+      position.push(result.position().x);
+      position.push(result.position().y);
+      return position;
```

Comment:
return something like {x : position.x, y: position.y}. This format will be better for later use

Reply:
Wouldn't `operatorCell.get('position')` work without creating a new element?

Reply:
fixed it.